cylc/flow/network/schema.py

```diff
@@ -88,7 +88,7 @@ def parse_node_id(item, node_type=None):
     """Parse definition, job, or proxy id argument returning components.
 
     Args:
-        item: An string representing a node ID. Jobs fill out
+        item (str): An string representing a node ID. Jobs fill out
             cycle%name%num first, cycle is irrelevant to Def
```

Comment:
I didn't update the delimiter `%` to `|` .. I suppose f-string doc (if possible?) would be inefficient (i.e. using `ID_DELIM`)

Reply:
Will fix the "An string" here too. Not sure about the delimiter and f-strings though, I'm a bit out of the loop (wasn't really using the code, just updated the docs)

Reply:
> Will fix the "An string" here too. Not sure about the delimiter and f-strings though, I'm a bit out of the loop (wasn't really using the code, just updated the docs)

Was just mentioning it while you're fixing my mistakes XD

Reply:
Oh, same error as above haha, thought this comment was on another line. Pushing updated commit.